### ISSUE TITLE
Dataset problem 

### ISSUE THREAD

**Issue by M11202226HSU** [mahmoudnafifi/Deep_White_Balance#16]
Since I couldn't find your [Rendered WB dataset], I decided to download the raw dataset. It contains 62,532 images and 1,991 ground truth images.
Could you please help me understand how to set up my dataset? I have a total of 62,532 images and 1,991 ground truth images. What should the data format be? Additionally, how should I divide the dataset into training, validation, and testing sets?

Thank you!

**Comment by mahmoudnafifi** [comment]
Hi, the dataset is available [here](https://github.com/mahmoudnafifi/WB_sRGB). If links are not working, please check the Google Drive mirror. 

Hope that helps.

**Comment by M11202226HSU** [comment]
Hello,

I have downloaded the dataset, and now I have input images and ground truth images. I noticed file names like 8D5U5524_C_FF, 8D5U5524_C_A4, 8D5U5524_C_AS, 8D5U5524_D_AS, 8D5U5524_C_L, and 8D5U5524_C_N. What do the suffixes C_FF, C_A4, C_AS, C_N, and C_L mean? Are they necessary for training?

Thank you for your response!

**Comment by mahmoudnafifi** [comment]
Yes they are necessary for training. File format is as follows: name_WB_picStyle and the corresponding ground truth image (for auto white balance): name_G_AS. G here refers to ground-truth (manually corrected white balance), C (for example) refers to cloudy white balance, and so on.

**Comment by M11202226HSU** [comment]
To train your model, should I place all the images (including 62,535 images) and ground truth (including 1,991 images) into one folder, or should I split them into training, validation, and testing sets? If splitting is required, what should the format be?

Thank you for your help !

**Comment by M11202226HSU** [comment]
I understand that C stands for cloudy and D stands for daylight, but I'm not sure about the following notations: C_"A4", C_"CS", C_"FF", C_"L", and C_"P". I think C_AS might represent the ground truth, but could you please explain the other labels?
Thank you for your response !!!

**Comment by mahmoudnafifi** [comment]
AS, CS, L, … are picture styles. 
Deep wb  performs three different wb settings (auto, indoor, outdoor) and thus we have three gt white balance (T, S, G: tungsten, shade, manual) with AS (as shot) picture style.

**Comment by M11202226HSU** [comment]
Should I simply place all the images and ground truth (GT) in the same folder for training, or is there a need to split them into training, validation, and testing sets?

Thank you for your help
